Lib/test/test_capi/test_pyrun.py

```diff
@@ -0,0 +1,47 @@
+import unittest, os
+from test.support import import_helper
+
+# Skip this test if the _testcapi module isn't available.
+_testcapi = import_helper.import_module('_testcapi')
```

Comment:
This is already covered by `Lib/test/test_capi/__init__.py`.

```suggestion
import _testcapi
```